app/jobs/import_content_item_job.rb

```diff
@@ -12,9 +12,32 @@ def perform(*args)
     content_id = args[0]
 
     attributes = content_items_service.fetch(content_id)
-    attributes = attributes.merge(content_items_service.links(content_id))
+    links = content_items_service.links(content_id)
```

Comment:
What about if our Jobs have a similar structure as our `Rake` tasks. Just one line of code calling a method within an importer.

```ruby
class ImportContentItemJob < ApplicationJob
  def perform(*args)
    content_id = args[0]
    Importers::SingleContentItem.import(content_id)
  end
``` 

This way we will be able to call this method within the console, for example. I know existing code is not designed this way, but feels to me like a good way to go.